numpy/core/__init__.py

```diff
@@ -41,20 +41,12 @@
 except ImportError as exc:
     import sys
     msg = """
-
-IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!
```

Comment:
Keeping this line gives us a nice filter for people who pasted the message without thinking, even if it is a little ugly. Happy to go either way on this

Reply:
restoring